docs/source/hpc-intro-text.rst

```diff
@@ -0,0 +1,23 @@
+**Definition:** HPC-stack is a repository that provides a unified, shell script-based build system for 
+building the software stack required for the `Unified Forecast System (UFS) <https://ufscommunity.org/>`_ and applications. 
+
+Background
+------------------------
+The UFS Weather Model draws on over 50 code libraries to run its applications. These libraries range from libraries developed in-house at NOAA (e.g. NCEPLIBS, FMS, etc.) to libraries developed by NOAA's partners (e.g. PIO, ESMF etc) to truly third party libraries (e.g. NETCDF). Individual installation of these libraries is not practical, so the `HPC-Stack <https://github.com/NOAA-EMC/hpc-stack>`_ was developed as a central installation system to ensure that the infrastructure environment across multiple platforms is as similar as possible. Installation of the HPC-Stack is required to run the SRW. 
```

Comment:
HPC-stack is model or system agnostic.
Please rephrase as to not give the opinion that HPC-stack was built specifically for the UFS weather model or JEDI.
You could say, HPC-stack was built to provide libraries and dependencies in a consistent manner for NWP applications.